test/sql/copy/s3/s3_hive_partition.test

```diff
@@ -88,13 +88,13 @@ COPY (SELECT * FROM t1) TO 's3://test-bucket/hive-partitioning/filter-test-csv'
 
 # There should be Table Filters (id < 50) and file filters (c = 500)
 query II
-EXPLAIN select a from parquet_scan('s3://test-bucket/hive-partitioning/filter-test-parquet/*/*.parquet', HIVE_PARTITIONING=1) where c=500 and a < 4;
+EXPLAIN select a from parquet_scan('s3://test-bucket/hive-partitioning/filter-test-parquet/*/*.parquet', HIVE_PARTITIONING=1, HIVE_TYPES_AUTOCAST=0) where c=500 and a < 4;
 ----
 physical_plan	<REGEX>:.*PARQUET_SCAN.*Filters:.*a<4 AND a IS NOT.*NULL.*File Filters: \(CAST\(c AS.*INTEGER\) = 500\).*
 
 # There should be Table Filters (id < 50) and file filters (c = 500)
 query II
-EXPLAIN select a from read_csv_auto('s3://test-bucket/hive-partitioning/filter-test-csv/*/*.csv', HIVE_PARTITIONING=1, columns={'a': 'INT', 'b':'INT', 'c':'INT'}) where c=500 and a < 4;
+EXPLAIN select a from read_csv_auto('s3://test-bucket/hive-partitioning/filter-test-csv/*/*.csv', HIVE_PARTITIONING=1, HIVE_TYPES_AUTOCAST=0, columns={'a': 'INT', 'b':'INT', 'c':'INT'}) where c=500 and a < 4;
```

Comment:
is it also possible to leave the default value for `HIVE_TYPES_AUTOCAST` (assuming it is 1) and change the expected result to 
`FILTER.*(a < 4).*READ_CSV_AUTO.*File Filters: \(c = 500\)`
or something similar?

Reply:
I set it to 1 in a few places, it shouldnt matter for the tests whether its 0 or 1 i think? important thing is that it's explicit so that changing the default setting doesnt break these tests

Reply:
aaaah, true. That makes sense